### PR TITLE
#12605: Add governor notes and move models steps into separate steps

### PR DESCRIPTION
**Pull request by tt-rkim**
### Ticket

#12605

### Problem description

@mbahnasTT requested instructions on perf governor for models.

### What's changed

Add separate models section for installation.

### Checklist
- [ ] Post commit CI passes
- [ ] Blackhole Post commit (if applicable)
- [ ] Model regression CI testing passes (if applicable)
- [ ] Device performance regression CI testing passes (if applicable)
- [ ] New/Existing tests provide coverage for changes

### CODE REVIEW

**Comment by tt-rkim** [comment]
@mbahnasTT Please review these changes for perf governor.

File: INSTALLING.md [hunk 1]
```diff
@@ -146,15 +146,23 @@ to install with `pip`:
...
 
-If you are going to try our pre-built models in `models/`, then you must also
-further install their required dependencies and environment variables:
+4. (For models users only) Set up environment for models
+
+If you are going to try our pre-built models in `models/`, then you must execute
+the following to:
+
+- install their required dependencies
+- set appropriate indenvironment variables
```
[the remark this answers is not in the document]
Comment (tt-rkim) [reply INSTALLING.md:155]: Yea it's not new spelling

[the remark this answers is not in the document]
Comment (tt-rkim) [reply INSTALLING.md:155]: fixed

**Comment by mbahnasTT** [comment]
Thanks @tt-rkim !

**Pull request merged**